api/config.py

```diff
@@ -9,10 +9,10 @@
 app.config[
     "SQLALCHEMY_DATABASE_URI"
 ] = "postgresql://{user}:{password}@{host}/{db}".format(
-    user=os.getenv("DB_USER", "admin"),
-    password=os.getenv("DB_PASS", "development"),
+    user=os.getenv("DB_USERNAME", "admin"),
+    password=os.getenv("DB_PASSWORD", "development"),
     host=os.getenv("DB_HOST", "pg"),
-    db=os.getenv("DB_NAME", "api_development"),
+    db=os.getenv("DATABASE_NAME", "api_development"),
```

Comment:
We pass the production variables in `docker-compose.yaml`, and there we're using different variable names -- this brings the two things in sync.